### DISCUSSION TITLE
Error: prisma:client:operation span is expected to be entered in the client extension when tracing is enabled

### DISCUSSION THREAD

**Discussion by harshsbhat** [prisma/prisma#24847] [Q&A]
### Question

I was trying to enable Prisma Optimize for cal.com. But its giving me the above error. No documentation or any resource tells me how to solve this issue. Can someone help?

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @harshsbhat 👋 

Thank you for raising this question. 

I have shared your question with the engineering team. Either I or a member of the engineering team will provide you with an update. 

It would also be helpful if you could share any steps to reproduce this as well.

**Reply by harshsbhat** [reply]
thanks @ludralph any response from the team?

**Reply by ludralph** [reply]
Hi @harshsbhat, An engineer has been assigned to work on this. I will get back to you with any updates I receive from the team.

**Reply by ludralph** [reply]
Hi @harshsbhat, Can you try installing this dev version to see if this issue is resolved for you?
```
"@prisma/extension-optimize: 0.0.0-dev.202407222340"
```

**Reply by harshsbhat** [reply]
Sure I will try it in sometime

**Reply by harshsbhat** [reply]
@ludralph Installing `"@prisma/extension-optimize: 0.0.0-dev.202407222340"` this worked

**Comment by brook-digi** [comment]
Facing a similar issue, any workarounds?

**Comment by ali-master** [comment]
Any update? I'm facing this issue

**Reply by ali-master** [reply]
When i install prisma open telemetry package and initiate it, the issue happens

**Reply by ludralph** [reply]
Hi @ali-master 

This issue has been fixed now. Can you try with the latest version of Prisma 6.2.1?

**Comment by tomshudev** [comment]
I'm having the same issue, tried the suggested dev version with no luck, it would really help us

**Reply by ludralph** [reply]
Hi @tomshudev 

This issue has been fixed now. Can you try with the latest version of Prisma 6.2.1

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!;

**Reply by harshsbhat** [reply]
Perfect. I think with the newer update this is pretty irrelevant anyways.

**Comment by bytemtek** [comment]
Still same error

**Comment by ankur-arch** [comment]
Hey @bytemtek 👋 , could you specify which version of Prisma ORM you're using?

**Reply by Pruxis** [reply]
We're using 6.4.1 with optimize 1.1.8 and facing the same issue

**Reply by Pruxis** [reply]
In a monorepo with the following client:
```
generator client {
    previewFeatures = ["relationJoins"]
    provider        = "prisma-client-js"
    output          = "../generated/client"
    binaryTargets   = ["native", "linux-musl-openssl-3.0.x"]
}
```

**Reply by ankur-arch** [reply]
Thanks @Pruxis. I've communicated this with the team.

**Reply by nurul3101** [reply]
@Pruxis I was able to get Optimize working in a monorepo setup. Here is a working example which might be helpful to you: https://github.com/nurul3101/monorepo-optimize

I do get some warnings which I have shared with team internally. The warning do not cause any issues, Optimize works as expected.
